benchmarks/benchmark_qt_viewer_image.py

```diff
@@ -79,11 +79,16 @@ def setup(self):
         _ = QApplication.instance() or QApplication([])
```

Comment:
Just out of curiosity, is there a particular reason this benchmark class name doesn't follow the *Suite convention of the others?

Reply:
no wasn't intentional. We can have another convo about whether we like `Suite` everywhere, but I think that can be on another PR. I've added it here

Reply:
Sounds good!